private/rules/jvm_import.bzl

```diff
@@ -15,27 +15,38 @@ def _jvm_import_impl(ctx):
 
     injar = ctx.files.jars[0]
     outjar = ctx.actions.declare_file("processed_" + injar.basename, sibling = injar)
-    ctx.actions.run_shell(
-        inputs = [injar] + ctx.files._add_jar_manifest_entry,
+    args = ctx.actions.args()
```

Comment:
Do you mind writing comments to explain why this is needed, especially for Bazel 4.0.0? (forcing the javac flag is another problem, but let's handle it separately)

Reply:
Done